src/game/etj_timerun_v2.cpp

```diff
@@ -1092,46 +1121,53 @@ void ETJump::TimerunV2::checkRecord(Player *player) {
           }
         }
 
+        // this seems redundant...? aren't these false by default anyway?
```

Comment:
Is it possible that result->isTopRecordPerSeason is not set for a season?

Reply:
I don't think so, that would mean that there are no active seasons I think, and since you can't delete default season, that is never the case.